test/validation/api/scheduler/scheduler.c

```diff
@@ -173,10 +177,11 @@ static void scheduler_test_wait_time(void)
 	queue = odp_queue_create("dummy_queue", &qp);
 	CU_ASSERT_FATAL(queue != ODP_QUEUE_INVALID);
 
-	wait_time = odp_schedule_wait_time(ODP_TIME_SEC_IN_NS);
+	wait_time = odp_schedule_wait_time(WAIT_TIMEOUT);
 	start_time = odp_time_local();
-	odp_schedule(&queue, ODP_SCHED_NO_WAIT);
+	ev = odp_schedule(NULL, ODP_SCHED_NO_WAIT);
 	end_time = odp_time_local();
+	CU_ASSERT(ev == ODP_EVENT_INVALID);
```

Comment:
Possible event is not freed.

Reply:
Fixed in v3. Changed to CU_ASSERT_FATAL as no event is expected.